lib/mayaUsd/ufe/UsdAttributes.cpp

```diff
@@ -126,89 +126,87 @@ Ufe::Attribute::Type
-                //// GfVec4f 		{SdfValueTypeNames->Color4d.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfVec4d 		{SdfValueTypeNames->Quath.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfQuath 		{SdfValueTypeNames->Quatf.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfQuatf 		{SdfValueTypeNames->Quatd.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfQuatd 		{SdfValueTypeNames->Matrix2d.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfMatrix2d 		{SdfValueTypeNames->Matrix3d.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfMatrix3d 		{SdfValueTypeNames->Matrix4d.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfMatrix4d 		{SdfValueTypeNames->Frame4d.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfMatrix4d 		{SdfValueTypeNames->TexCoord2f.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfVec2f 		{SdfValueTypeNames->TexCoord2d.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfVec2d 		{SdfValueTypeNames->TexCoord2h.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfVec2h 		{SdfValueTypeNames->TexCoord3f.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfVec3f 		{SdfValueTypeNames->TexCoord3d.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfVec3d 		{SdfValueTypeNames->TexCoord3h.GetHash(), Ufe::Attribute::kUnknown},
-                //// GfVec3h
-                // To add?
-                // SdfValueTypeName BoolArray;
-                // SdfValueTypeName UCharArray, IntArray, UIntArray, Int64Array, UInt64Array;
-                // SdfValueTypeName HalfArray, FloatArray, DoubleArray;
-                // SdfValueTypeName StringArray, TokenArray, AssetArray;
-                // SdfValueTypeName Int2Array,     Int3Array,     Int4Array;
-                // SdfValueTypeName Half2Array,    Half3Array,    Half4Array;
-                // SdfValueTypeName Float2Array,   Float3Array,   Float4Array;
-                // SdfValueTypeName Double2Array,  Double3Array,  Double4Array;
-                // SdfValueTypeName Point3hArray,  Point3fArray,  Point3dArray;
-                // SdfValueTypeName Vector3hArray, Vector3fArray, Vector3dArray;
-                // SdfValueTypeName Normal3hArray, Normal3fArray, Normal3dArray;
-                // SdfValueTypeName Color3hArray,  Color3fArray,  Color3dArray;
-                // SdfValueTypeName Color4hArray,  Color4fArray,  Color4dArray;
-                // SdfValueTypeName QuathArray,    QuatfArray,    QuatdArray;
-                // SdfValueTypeName Matrix2dArray, Matrix3dArray, Matrix4dArray;
-                // SdfValueTypeName Frame4dArray;
-                // SdfValueTypeName TexCoord2hArray, TexCoord2fArray, TexCoord2dArray;
-                // SdfValueTypeName TexCoord3hArray, TexCoord3fArray, TexCoord3dArray;
-            };
+    static const std::unordered_map<size_t, Ufe::Attribute::Type> sUsdTypeToUfe {
```

Comment:
I really hate this formatting and this make it impossible to read. Can you please revert this map back to its original state **before the first** clang-format was applied. Then you can manually change the tabs to spaces. And add in the clang-format off/on comment so this block will remain unchanged by clang-format.

Reply:
Removed dead code and fixed based on our discussion.

Reply:
Much better, thanks.